login/login_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package login_test
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/juju/cmd/cmdtesting"
+	jc "github.com/juju/testing/checkers"
+	"github.com/juju/usso"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/identity/login"
+)
+
+type cliSuite struct {
```

Comment:
Not sure "cliSuite" is that appropriate a name any more, but I can't think of a better one currently :)